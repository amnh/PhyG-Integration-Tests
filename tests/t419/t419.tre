(AlbanianTosk:0.0,(OldPrussian:38.3,(Irish:4.6,(Manx:5.1,((ArmenianWestern:23.0,Cornish:17.0)HTU17:41.6,(EasternFrisianSaterland:5.0,((KolschColognian:5.0,(Icelandic:56.0,(Yiddish:20.0,(Faroese:25.0,Scots:10.0)HTU20:52.2)HTU19:50.2)HTU21:21.3)HTU22:43.0,(Assamese:24.6,(SanskritVedic:12.0,(AncientIonicGreek:24.2,Danish:37.0)HTU25:24.0)HTU24:10.1)HTU23:4.6)HTU26:0.0)HTU28:75.0)HTU18:33.9)HTU27:13.0)HTU30:0.1)HTU29:0.0)HTU16[1179.9];
