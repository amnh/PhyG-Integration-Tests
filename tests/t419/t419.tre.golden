(AlbanianTosk:0.0,(OldPrussian:38.3,(Manx:4.1,((ArmenianWestern:13.0,Cornish:27.0)HTU17:42.0,(Irish:2.9,((EasternFrisianSaterland:6.9,((Yiddish:43.2,((Assamese:33.7,SanskritVedic:14.0)HTU20:15.0,(AncientIonicGreek:24.2,Danish:38.0)HTU28:13.0)HTU24:12.0)HTU23:3.7,(Icelandic:75.3,KolschColognian:7.0)HTU30:34.0)HTU21:6.3)HTU25:34.2,(Faroese:26.0,Scots:11.0)HTU26:32.2)HTU27:4.5)HTU18:44.0)HTU29:12.5)HTU19:23.1)HTU22:0.0)HTU16[1111.0000000000002];
